src/sos/workflow_executor.py

```diff
@@ -1904,7 +1903,7 @@ def run_as_nested(self,
                 #     break
                 else:
                     time.sleep(0.01)
-        except KeyboardInterrupt:
+        except KeyboardInterrupt as exec_error:
             if exec_error.errors:
```

Comment:
NO. `exec_error` is defined somewhere else and you are overwriting the variable here.

Why cannot you start with simpler warnings that  I listed yesterday?